common/on_actions/eaw_on_actions.txt

```diff
@@ -5,10 +5,26 @@ on_actions = {
 		effect = {
 			log = "[GetDateText] Equestria at War main on_startup begin"
 			every_state = {
-				limit = {
-					is_on_continent = europe #Equestria
+				if = {
+					limit = { in_dry_subarctic_climate_region = yes }
+					set_variable = { EaW_state_winter_grade_var = 5 }
```

Comment:
We are going to have to discuss these changes. What they are. And what impact they have on the game.

Reply:
Trying to understand the impact of broken mechanics should be fun.